transports/rabbitmq/delayed-delivery.md

```diff
@@ -21,58 +21,111 @@ N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.destination
 
 Where `N` is either `0` or `1`, representing the delay value in binary, and `destination` is the name of endpoint the delayed message will be sent to.
 
+As an example, a delay of 10 seconds (`1010` in binary) on a message bound for the `destination` queue would be encoded with a routing key of:
+
+```
+0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.1.0.1.0.destination
+```
+
 
 ### Delay levels
 
 Each exchange/queue pair that makes up a level represents one bit of the total delay value. By having 28 of these levels, corresponding to `2^27` through `2^0`, the delay infrastructure can delay a message for any value that can be represented by a 28-bit number. With 28 total levels, the maximum delay value is 268,435,455 seconds, or about 8.5 years.
 
 A delay level is created by declaring a topic exchange that is bound to a queue with a routing key of `1`, and to the exchange corresponding to `level - 1` with a routing key of `0`. The queue for the delay level is declared with an [`x-message-ttl`](https://www.rabbitmq.com/ttl.html) value corresponding to `2^level` seconds. The queue is also declared with an [`x-dead-letter-exchange`](https://www.rabbitmq.com/dlx.html) value corresponding to the `level - 1` exchange, so that when a message in the queue expires, it will be routed to the `level - 1` exchange.
 
+The delay levels are connected in this manner, from highest (27) to lowest (0). Each delay level's routing key's add wildcards as needed so that they are looking at the portion of the message's routing key that corresponds to its delay level.
+
+The following diagram illustrates the relationships between exchanges and queues, which are connected by bindings and `x-message-ttl` delay values. It's important to note the wildcard rules for RabbitMQ binding expressions, where *word* describes a dot-delimited segment:
+
+* `*` substitutes for exactly one word
+* `#` substitutes for 0 or more words
+
+<style>
+.edgeLabel {
+    font-family: Consolas, Courier New, monospace;
+}
+</style>
+
 ```mermaid
 
 graph TD
 
-subgraph Delay Level
+subgraph Delay Level Exchanges, Queues, Bindings, and TTLs
 
-exchangeN(Level N)
-queueN[TTL 2^N]
-exchangeN-1(Level N-1)
+exchange27(Exchange: nsb.delay-level-27)
+queue27[Queue: nsb.delay-level-27]
+exchange26(Exchange: nsb.delay-level-26)
+queue26[Queue: nsb.delay-level-26]
+exchange25(Exchange: nsb.delay-level-25)
 
-exchangeN -->|bit=1| queueN
-exchangeN -->|bit=0| exchangeN-1
-queueN -->|expired<br>message| exchangeN-1
+hiddenExchanges(Not Shown: nsb.delay-level-24 through nsb.delay-level-2)
 
-classDef exchangeClass stroke:#000000,stroke-width:2px;
-class exchangeN,exchangeN-1 exchangeClass
+exchange1(Exchange: nsb.delay-level-01)
+queue1(Queue: nsb.delay-level-01)
+exchange0(Exchange: nsb.delay-level-00)
+queue0(Queue: nsb.delay-level-00)
+delivery(Exchange: nsb.delay-delivery)
+
+exchange27 -->|"Binding: 1.#<br/>(No * wildcards)"| queue27
```

Comment:
Is the listing of how many wildcards there are really needed? Seems like it's just noise and taking up space.

Reply:
Without it, not being in a table or right next to each other, makes it difficult to see that they're getting bigger.

Reply:
I'll wait and see how it looks on the page.